src/library/Uno.Material/Styles/Controls/v2/CommandBar.xaml

```diff
@@ -114,55 +65,37 @@
 		</Grid>
 	</ControlTemplate>
 
-	<Style x:Key="MaterialBaseCommandBarStyle"
-		   TargetType="CommandBar">
+	<!-- Define the base style for Material CommandBar -->
+	<Style x:Key="MaterialBaseCommandBarStyle" TargetType="CommandBar">
 		<!-- Setting content to empty to avoid getting the datacontext -->
 		<Setter Property="Content" Value="" />
 		<Setter Property="ContentTemplate">
 			<Setter.Value>
 				<DataTemplate>
-					<TextBlock Text="{Binding}"
-							   Style="{StaticResource MaterialTitleLarge}" />
+					<TextBlock Text="{Binding}" Style="{StaticResource MaterialTitleLarge}" />
 				</DataTemplate>
 			</Setter.Value>
 		</Setter>
```

Comment:
I think we can remove these now, we should add a CommandBar sample page if we don't already have one and test what this looks like without these Setters